src/pages/List/components/Product/Product.tsx

```diff
@@ -0,0 +1,24 @@
+import { ProductType } from '@/types'
+
+import { useProduct } from './hooks'
+
+const Product = ({ id, imageUrl, name, price }: ProductType) => {
```

Comment:
Props 타입을 별도로 선언해주시면 다른 prop 을 전달받을때에도 확장하기 용이할 것 같아요

```suggestion
const Product = ({ product}: { product: ProductType}) => {
```

Reply:
[f4edf65](https://github.com/next-step/react-shopping-cart/pull/2/commits/f4edf652308f40859c902730ac5307656edcd79b)